frontend/src/metabase/components/SaveQuestionForm/context.tsx

```diff
@@ -53,6 +54,8 @@ export const SaveQuestionProvider = ({
   onCreate,
   onSave,
   multiStep = false,
+  withCollectionPicker = true,
```

Comment:
Collection Picker should always be present unless the SDK user explicitly disables it in their `saveOptions`. This is not applicable in the main Metabase app currently.